docs/copied-from-beats/shared-logstash-config.asciidoc

```diff
@@ -39,16 +39,23 @@ Beats connections.
 For this configuration, you must <<load-template-manually,load the index template into Elasticsearch manually>>
 because the options for auto loading the template are only available for the Elasticsearch output.
 
-ifdef::allplatforms[]
+ifndef::win-only[]
+ifndef::apm-server[]
 
 include::../../libbeat/docs/step-test-config.asciidoc[]
 
-endif::allplatforms[]
+endif::apm-server[]
+ifdef::apm-server[]
 
-ifdef::win[]
+include::./step-test-config.asciidoc[]
```

Comment:
[Link](https://www.elastic.co/guide/en/apm/server/6.5/logstash-output.html). Need to determine which (if either) TIP to include here: https://github.com/elastic/apm-server/pull/1634/files#diff-7efd36d63ce5c77977dc56684e2363e7R1

Reply:
_i think_ it should be `require sudo = false`, @graphaelli can you confirm me this?
for reference we have https://github.com/elastic/apm-server/issues/1401 open, which is an issue with the default user, but apm-server does not require sudo per se.

Reply:
agreed, no sudo needed